packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -263,6 +263,17 @@ export const transformContentTypesToModels = (contentTypes: LoadedContentTypeMod
       },
     };
 
+    // Add indexes to model
+    if (contentType.modelType === 'contentType') {
+      model.indexes = [
+        {
+          name: identifiers.getUniqueIndexName([contentType.collectionName, 'document_id']),
```

Comment:
```suggestion
          name: identifiers.getUniqueIndexName([contentType.collectionName, 'document_id']),
```
 in theory this would be `[contentType.collectionName, 'document_id', 'locale', 'published_at']` but tbh that could potentially reintroduce an issue with long names at some point 😆

But maybe just call the second param 'documents' so it's more clear that this is unique on documents rather than a specific column

Reply:
@Marc-Roig I just noticed, this is using `getUniqueIndexName()` but it looks like this is just a regular index.

Reply:
you are right, will use a regular index name